MergerService/Runners/TaskExecutor.cs

```diff
@@ -161,7 +153,7 @@ public void ExecuteTask(MergeTask task, ITaskUtils taskUtils, string? managerCal
                                     // Create tile builder list for current coord for all sources
                                     List<CorrespondingTileBuilder> correspondingTileBuilders = new List<CorrespondingTileBuilder>();
                                     // Add target tile
-                                    correspondingTileBuilders.Add(() => getTileByCoord(sources[0], coord));
+                                    correspondingTileBuilders.Add(() => sources[0].GetCorrespondingTile(coord, shouldUpscale));
```

Comment:
why do you need to declare the "shouldUpscale" variable? why dont you just put the '!metadata.IsNewTarget' as this method arg?